packages/remix/test/integration/package.json

```diff
@@ -7,12 +7,15 @@
     "start": "remix-serve build"
   },
   "dependencies": {
+    "@fastify/formbody": "^7.4.0",
     "@mcansh/remix-fastify": "^3.2.2",
     "@remix-run/express": "1.17.0",
     "@remix-run/node": "1.17.0",
     "@remix-run/react": "1.17.0",
     "@remix-run/serve": "1.17.0",
     "@sentry/remix": "file:../..",
+    "express": "^4.19.2",
```

Comment:
Suggesting to have `express` explicitly listed as a dependency in this package. The fact that the upper level dev package is powered by `express` may not be true in the future. Plus it makes it clear, adapters for which frameworks are being covered here.

Reply:
That makes sense to me 👍